src/diffusers/schedulers/scheduling_flow_match_euler_discrete.py

```diff
@@ -212,6 +214,9 @@ def scale_noise(
         while len(sigma.shape) < len(sample.shape):
             sigma = sigma.unsqueeze(-1)
 
+        if noise is None:
```

Comment:
can we `noise` a required arg instead? 
to properly generate noise inside scale_noise we need to pass a generator too, I don't think we need to do that